lib/taskana-history-example/.gitignore

```diff
@@ -0,0 +1,29 @@
+*/target/
```

Comment:
why do we need multiple gitignore files? One should suffice

Reply:
You are right this is a copy/paste issue

Reply:
done